src/main/java/org/isf/medicals/manager/MedicalBrowsingManager.java

```diff
@@ -59,6 +59,17 @@ public class MedicalBrowsingManager {
 	public Medical getMedical(int code) throws OHServiceException {
 		return ioOperations.getMedical(code);
 	}
+	
+	/**
+	 * Returns the requested medical.
+	 *
+	 * @param code the medical prod_code.
```

Comment:
```suggestion
	 * @param prod_code the medical prod_code.
```
make it match the parameter in the method signature